packages/react-components/review/react-components.api.md

```diff
@@ -347,6 +347,23 @@ export interface DevicesButtonStyles extends ControlBarButtonStyles {
     menuStyles?: Partial<DevicesButtonContextualMenuStyles>;
 }
 
+// @internal
+export const _DrawerSurface: (props: _DrawerSurfaceProps) => JSX.Element;
+
+// @internal
+export interface _DrawerSurfaceProps {
```

Comment:
I notice a bunch of props you are making have flat bar naming. is there a reason for this?